File: C/linearSearch.c
// Linear Search
#include <stdbool.h>
#include <stdio.h>

void main() {
  int ar[] = {50,20,16,34,22,5};
  int len = sizeof(ar)/sizeof(int), key;
  bool found = false;
  
  scanf("%d",&key);
  
  for (int i=0;i<len;i++)
  {
    if (key == ar[i])
    {
      printf("Found at index %d",i);
      found = true;
      break;
    }
  }
  
  if (found == false)
    printf("Did not find");
}

File: Python/linearSearch.py
def linear(l,item):
    if item in l:
        for i in l:
            if i == item:
                indx = l.index(item)
                print("Item",item,"found at index",indx)
    else:
        print("Item not found in list")

def traverse(l):
    print("Traversing the list: ")
    print(l[:])

def main():
  item = int(input("Enter item to search: "))
  l = map(int,input().split())
  linear(l,item)
  
if __name__ == '__main__':
  main()
